src/app/workspace/organization/users/list/organization-user-list.component.html

```diff
@@ -0,0 +1,35 @@
+<table>
+  <thead class="thead-light medium-text">
+  <tr>
+    <td colspan="3">
+      <organization-user-filter>
+      </organization-user-filter>
+    </td>
+    <td class="px-2"
+        i18n="@@organizations.user.roles.create">
+      Create Project
+    </td>
+    <td class="px-2"
+        i18n="@@organizations.user.roles.manage">
+      Manage Organization
+    </td>
+    <td colspan="3">
+    </td>
+  </tr>
+  </thead>
+  <tbody>
+  <ng-container *ngFor="let user of users">
```

Comment:
Should not this be `<tr>` element?

Reply:
No. `organization-user` and `organization-user-groups` are `<tr>`-like elements.